pkg/plugins/golang/v4/scaffolds/internal/templates/api/types.go

```diff
@@ -89,6 +90,23 @@ type {{ .Resource.Kind }}Spec struct {
 type {{ .Resource.Kind }}Status struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
```

Comment:
should doc be more explicit here?
```suggestion
	// Important: Run "make generate manifests" to regenerate code after modifying this file
```

Reply:
sure, it can be follow up